compiler/il/OMRBlock.cpp

```diff
@@ -1446,10 +1446,10 @@ OMR::Block::splitPostGRA(TR::TreeTop * startOfNewBlock, TR::CFG *cfg, bool copyE
          // For all the nodes in the nodeTable for which we can not find replacement yet, create replacement node using available registers.
          if (!iter->second.second)
             {
-            TR::SymbolReference *ref = value->getOpCode().isLoadReg() ? value->getRegLoadStoreSymbolReference() : createSymRefForNode(comp, methodSymbol, value, self()->getExit());
             std::pair<TR_GlobalRegisterNumber,TR_GlobalRegisterNumber> regInfo = findAvailableRegister(comp, iter->first, unavailableRegisters);
             if (regInfo.first > -1)
```

Comment:
@andrewcraik  Actually I can not move the code to create GlRegDeps for exit and entry node here. This one handles nodes for which we do not have replacement yet. Other nodes like `regLoad` which required to be uncommoned after split point will not enter this condition as for those node, a replacement node would just be a copy of the original `regLoad`.

Reply:
Changes we were discussing in https://github.com/eclipse/omr/pull/4551#discussion_r344807984 can not be done, as second  `reg` case if-else handles all the nodes which are stored into register which includes already existing `regLoad`